### PR TITLE
Make fileActions mixin standalone again.

### PR DESCRIPTION
**Pull request by dschmidt**
## Description
In #6445 we introduced a dependency on isFilesAppActive for the fileActions mixin. We added it to
ContextActions and BatchActions, but there are way more places where the mixin is used. That's why
we add a computed to the mixin itself now. Because mixins belong to the options api we unfortunately
cannot use composables directly and extract helper functions for that reason.

## Related Issue

- Fixes <issue_link>

## How Has This Been Tested?

- test environment:
- test case 1:
- test case 2:
- ...

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Technical debt
- [ ] Tests

## Checklist:

- [ ] Code changes
- [ ] Unit tests added
- [ ] Acceptance tests added
- [ ] Documentation ticket raised: <link> 

## Open tasks:

- [ ] ...

### CODE REVIEW

File: packages/web-app-files/src/components/AppBar/SelectedResources/BatchActions.vue [hunk 1]
```diff
@@ -44,6 +39,11 @@ export default {
     Move,
     Restore
   ],
+  setup() {
+    return {
+      isFilesAppActive: useIsFilesAppActive()
```
Comment (kulmann) [inline packages/web-app-files/src/components/AppBar/SelectedResources/BatchActions.vue:44]: I'm confused, can't you delete this again with the helper function now being used in the mixins?

File: packages/web-app-files/src/mixins/fileActions.js [hunk 2]
```diff
@@ -55,6 +58,12 @@ export default {
     ...mapGetters('Files', ['highlightedFile', 'currentFolder']),
     ...mapGetters(['capabilities', 'configuration']),
 
+    // TODO: Replace this with the useIsFilesAppActive composable
+    // once we port this mixin to something using the composition api
+    isFilesAppActive() {
```
Comment (kulmann) [inline packages/web-app-files/src/mixins/fileActions.js:63]: Oh wow, is it really sufficient to provide the function here? Are we using the `fileActions` mixin really in all locations that use any of the specific file action mixins?

Comment (dschmidt) [reply packages/web-app-files/src/mixins/fileActions.js:63]: That's why the composable is still used in BatchActions ... 
We include single actions there and not the fileActions mixin.

... but that's a very good point, this has become hard to reason about. I will add this to all the mixins :(

**Comment by ownclouders** [comment]
Results for <strong>oCISSharingInternal2</strong> https://drone.owncloud.com/owncloud/web/22895/60/1
The following scenarios passed on retry:
- webUISharingInternalUsers/shareWithUsers.feature:342

**Review by kulmann — APPROVED** [review]
👍

**Pull request merged**